src/Symfony/Component/HttpKernel/Resources/welcome.html.php

```diff
@@ -2,123 +2,253 @@
-            </div>
-            <h1><small>Welcome to</small> Symfony <span class="version"><?= $version; ?></span></h1>
-        </div>
+    <header>
+        <div class="wrapper">
+            <section class="logo">
+                <?= renderSymfonyLogoSvg() ?>
+                <h1>
+                    <small>Welcome to</small>
+                    <span translate="no" class="notranslate">Symfony</span> <?= explode('.', $version, 2)[0] ?>
+                </h1>
+            </section>
 
-        <div class="status">
-            <code>
-                <span class="check">
-                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M0 0h24v24H0z" fill="none"/><path d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"/></svg>
-                </span>
-                <span><?= $projectDir; ?></span>
-            </code>
-            <p class="status-ready">Your application is now ready and you can start working on it.</p>
+            <section class="checks">
+                <ul>
+                    <li>
+                        <span><?= renderCheckIconSvg() ?></span>
+                        <span>You are using Symfony <strong><?= $version ?></strong> version</span>
+                    </li>
+
+                    <li>
+                        <span><?= renderCheckIconSvg() ?></span>
+                        <span>Your application is ready at: <code translate="no" class="notranslate project_dir_path"><?= $projectDir ?></code></span>
```

Comment:
i'd make this the first list-item, the major version is already on top in heading1